nexus/src/app/rack.rs

```diff
@@ -21,6 +23,7 @@ impl super::Nexus {
     pub(crate) fn as_rack(&self) -> db::model::Rack {
         db::model::Rack {
             identity: self.api_rack_identity.clone(),
+            initialized: true,
```

Comment:
Out of curiosity, why is this `true`? (Not suggesting it's wrong, I just don't have context for this method)